sdk/core/Azure.Core/feature_lifecycle.md

```diff
@@ -0,0 +1,31 @@
+# Core Ship Sequence (draft)
+Legend:
+- diamond shapes == Phase
+- hexagonal shapes == Sub Step
+
+```mermaid
+flowchart TD
+    A{Feature Definition} --> B{{Feature Triage}}
+    B -->|Committed| SelS{{Select Shipping Sprint}}
+    SelS --> AtS{{Add Issue to Sprint Plan}}
+    AtS --> DI{Design and<Br/> Implementation}
+    DI --> DDes{{Document Design}}
+    DDes --> Cost{{Costing of Feature}}
+    Cost --> DDec{{Document Decisions}}
+    DDec --> AP{Approval Phase}
+    AP --> Arch{{Arch Board}}
+    Arch --> DocCh{{Document any Changes}}
+    DocCh --> ApplyFB{{Apply Feedback}}
+    ApplyFB --> Merge{{Merge to Main}}
+    Merge --> FinalApp{{Final APIView Approval}}
+    FinalApp --> RP{Release Phase}
+    RP --> Ship{{Ship Package}}
+    Ship --> Refs{{Fixup Core References}}
```

Comment:
We're working with @AlexGhiondea on an approach to DPG where there is never a project reference to Core from DPG clients -- they can only ever depend on a released version of Azure.Core. Would we want to take the same approach for existing HLCs? If so, the "Fixup Core References" and "Dependent Clients Ready to Ship" steps would go away. If not, I'd be interested in understanding how HLC (Gen 1 HLC) and DPG differ in this case, because I think it will be interesting as we consider how DPGs evolve into Gen 2 HLCs.